SimPEG/EM/Base.py

```diff
@@ -390,39 +394,26 @@ def MfRho(self):
             self._MfRho = self.mesh.getFaceInnerProduct(self.rho)
         return self._MfRho
 
-    @property
-    def MfRhoDerivMat(self):
```

Comment:
similar comment.

Reply:
`_MfRhoDeriv` is cleared on any updates to sigma. See line 82 above